src/librustdoc/html/static/main.js

```diff
@@ -83,12 +84,20 @@ function getSearchElement() {
     return document.getElementById("search");
 }
 
+function getThemesElementId() {
```

Comment:
Is there a reason this and `getThemePickerElementId` are functions instead of variables?

Reply:
That's beause we can't use constant that I turned it into a function. But I can use a variable. Which do you prefer?

Reply:
Script local variables seem better, to avoid thinking they need computing at all.

Reply:
Fine by me!